lab/docker/importer/volume/database/dbInit/1-pgsqlPermissions.sql

```diff
@@ -0,0 +1,4 @@
+
+CREATE USER clinlims PASSWORD 'clinlims';
+CREATE USER admin SUPERUSER PASSWORD 'cPATaahXJlSj';
```

Comment:
@mozzy11 Should we be exposing the password here?

Reply:
also, we have other passwords in the common.properties file - would it be possible to store the configuration data there? or is there a reason for this separate password setting?